tests/test_operations.py

```diff
@@ -642,7 +642,7 @@ def test_set_metric_value(self):
         my_operation_descriptor = self.sdc_device.mdib.descriptions.coding.get_one(coding, allow_none=True)
 
         operation_handle = my_operation_descriptor.Handle
-        for value in (Decimal(1), Decimal(42)):
+        for value in (Decimal(1), Decimal(42), 1.5, 10, "12"):

```

Comment:
so apparently you not get this floating point issue for `1.5`. I suggest changing it of of `1.1`, `1.2`, `1.3`, `1.4`, `1.6`, `1.7`, `1.8`, `1.9`, because you want to test this explicitly